go/arrow/arrio/arrio_test.go

```diff
@@ -94,19 +100,17 @@ func TestCopy(t *testing.T) {
 							mem := memory.NewCheckedAllocator(memory.NewGoAllocator())
 							defer mem.AssertSize(t, 0)
 
-							f, err := ioutil.TempFile("", "arrow-ipc-")
+							f, err := ioutil.TempFile(tempDir, "arrow-ipc-")
```

Comment:
```suggestion
							f, err := ioutil.TempFile(tempDir, "arrow-io-")
```

or just use the test name?